src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,464 @@
+    Default is :math:`(a, b)=(0.5, 0.05, 0.5, 0.05)`.
+    This implementation includes the Jacobian :math:`J_f` of :math:`f`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(2, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`2`-dimensional mean vector and
+        :math:`2 \times 2`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float, float, float, float), optional
+        Parameters :math:`(a, b, c, d)` for the logistic IVP.
+        Default is :math:`(a, b, c, d)=(0.5, 0.05, 0.5, 0.05)`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the logistic IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return lv_rhs(t, y, params)
+
+    def jac(t, y):
+        return lv_jac(t, y, params)
+
+    return IVP(timespan, initrv, rhs, jac)
+
+
+def lv_rhs(t, y, params):
+    """RHS for Lotka-Volterra."""
+    a, b, c, d = params
+    y1, y2 = y
+    return np.array([a * y1 - b * y1 * y2, -c * y2 + d * y1 * y2])
+
+
+def lv_jac(t, y, params):
+    """Jacobian for Lotka-Volterra."""
+    a, b, c, d = params
+    y1, y2 = y
+    return np.array([[a - b * y2, -b * y1], [d * y2, -c + d * y1]])
+
+
+def seir(timespan, initrv, params=(0.3, 0.3, 0.1, 1e7)):
+    r"""Initial value problem (IVP) based on the SEIR model.
+
+    The SEIR model with no vital dynamics is defined through
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            \frac{-\beta y_1 y_3}{N} \\
+            \frac{\beta y_1 y_3}{N} - \alpha y_2 \\
+            \alpha y_2 - \gamma y_3 \\
+            \gamma y_3
+        \end{pmatrix}
+
+    for some parameters :math:`(\alpha, \beta, \gamma, N)`.
+    N is a constant population count.
+    Default is :math:`(\alpha, \beta, \gamma, N)=(0.3, 0.3, 0.1, 10^7)`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(4, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`4`-dimensional mean vector and
+        :math:`4 \times 4`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float, float, float, float), optional
+        Parameters :math:`(\alpha, \beta, \gamma, N)` for the SEIR model IVP.
+        Default is :math:`(\alpha, \beta, \gamma, N)=(0.3, 0.3, 0.1, 10^7)`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the SEIR model IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return seir_rhs(t, y, params)
```

Comment:
according to codecov, this is not tested. Any idea why?

Reply:
There does not exist a test case for that. As of now, we do not test evaluations for IVPs for which no Jacobian is implemented. Shall I write according tests?

Reply:
I wrote a corresponding comment in the tests. Bottom line: that would be great, I cant believe we havent done this yet.